lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -1454,6 +1459,7 @@ class LineTouchData extends FlTouchData<LineTouchResponse> with EquatableMixin {
     LineTouchTooltipData? touchTooltipData,
     GetTouchedSpotIndicator? getTouchedSpotIndicator,
     double? touchSpotThreshold,
+    CalculateTouchDistance? distanceCalculator,
```

Comment:
You added this input parameter in `copyWith` function. But you didn't use it in the body.